server/src/utils/cosmosClient.test.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
```

Comment:
can this be src/db/cosmosClient.test.ts

Reply:
The tests are not pushed since the implementation has been shifted that I decided to fix the test after agreement.